CONTRIBUTING.md

```diff
@@ -44,25 +44,27 @@ follows:
      to the [Ruby style
      guide](https://github.com/SUSE/style-guides/blob/master/Ruby.md).
 
-  4. Make sure your change didn't break anything by building the RPM package
+  4. Update the package version (usually by `rake version:bump`) and add
+     a new entry into the `*.changes` file. For bigger changes or changes which
```

Comment:
Update the package version (in package/\*.spec, usually by ...) ... into the package/*.changes file.